certification/internal/shell/engine.go

```diff
@@ -126,14 +129,20 @@ func (e *PolicyEngine) GetContainerFromRegistry(containerLoc string, logger *log
 	}
 	lines := strings.Split(string(stdouterr), "\n")
 
+	tempdir, err := os.MkdirTemp(os.TempDir(), "preflight-*")
```

Comment:
Is this supposed to be `os.Mkdir`? I'm getting `os.MkdirTemp` doesn't exist

Reply:
Correct.

Reply:
@bcrochet yup, let's do that.